Add data use and access Guideline (#49)
ghga-de/docs.ghga.de: user_docs/user_stories/dua-guideline.md

# Data Use and Access Guidelines

## Purpose of this Article

The purpose of this document is to provide <general:Research Data Controller (RDC)|Research Data Controllers> with guidance on how to make decisions about <general:Data Access Request (DAR)> for data they have archived in GHGA. The document identifies ethical and organisational standards regarding operating procedures and decision-making criteria.

Please refer to the [glossary](../glossary.md) for definitions of capitalised terms.

## Introduction

<general:Research Data> such as <general:Omics Data> and other forms of genetic and health data used for scientific research purposes is considered sensitive personal data (special category data under Art. 9 GDPR) due to its informational content and the inherent risk of re-identification associated with it. A whole genome sequence is unique to an individual and so has the potential to differentiate them from the wider population. Moreover, the combination of Omics Data with clinical data in research carries further informational risks for the identifiable person as information about their current and future health could also be attributed to them.

> <general:Research Data> that is archived by GHGA for secondary use in scientific research is therefore subject to controlled access (see GHGA [Terms of Use](https://zenodo.org/records/11146387)). As part of data submission to GHGA, the <general:Data Submitters> should therefore implement a review process for <general:Data Access Request (DAR)|Data Access Requests> to ensure compliance with legal and ethical standards, especially to protect individual privacy.

Reviewer: Would suggest we say 'are therefore advised to implement a review process'.

Author: ```suggestion
<general:Research Data> that is archived by GHGA for secondary use in scientific research is therefore subject to controlled access (see GHGA [Terms of Use](https://zenodo.org/records/11146387)). As part of data submission to GHGA, the <general:Data Submitters> are therefore advised to implement a review process for <general:Data Access Request (DAR)|Data Access Requests> to ensure compliance with legal and ethical standards, especially to protect individual privacy.
```
👍🏻

   end
   rect rgb(250, 250, 250)
   note over DR, GHGA: Decision Implementation / Data Download
   RDC ->> GHGA : Informs GHGA on decision on  DAR<br/> (Acceptance / rejection)
   GHGA ->> DR : Informs DR on decision. <br/> If positive provides download link
   end
```

The primary objective of DACs is to oversee the access to Research Data. In contrast to <general:Research Ethics Committee|Research Ethics Committees> (Institutional Review Boards) that assess the full range of ethical issues associated with research projects, a DAC usually evaluates whether the proposed use of data is compatible with the data use conditions, in particular those specified in the data subject’s consent.

We recommend that DACs authorising access to data deposited with GHGA establish a set of documents to serve as a basis for them to operate and reach decisions. These will usually include terms of reference (TOR) and standard operating procedures (SOP) for the DAC as well as a list of criteria for data access decisions and a <general:Data Transfer Agreement (DTA)> template. These documents are further described below.

> ## Key Documents Recommended for DAC Operation

Reviewer: Should we add links to the documents? For example Line 84 references the SOPs, so should this be an internal link?

Author: But this would not be an SOP in our GHGA internal documentation but something the controller / DAC has to maintain. In general we have none of these documents.

The TOR should specify:

* The role and purpose of the DAC;

* The relationship between the DAC and the Research Data Controller (e.g.: Is the DAC part of the data-controlling institution or has an external DAC been appointed by the Research Data Controller?);

* The particularities of membership (e.g.: How many permanent and associate members does the DAC have? Who are the members and what are their areas of expertise? How are members appointed?);

* A description of the different roles and responsibilities within the DAC, if applicable (e.g.: Chair, Co-chair);

* The period of service of the DAC and/or its members;

* The nature and structure of DAC meetings and communication (as specified in more detail in the SOP);

* The process by which other parties at the Research Data Controller’s institution can be involved, e.g. the data protection officer or the legal department.

> ### DAC Standard Operating Procedures (SOP)

Reviewer: I do feel it makes more sense to SOPs rather than SOP.

Author: ```suggestion
### DAC Standard Operating Procedures (SOPs)
```

The standard operating procedures (SOP) define how the routine tasks of the DAC should be carried out and help DAC members to work in an efficient and consistent manner. The SOP should be documented and subject to change through a predefined process (see last bullet point of the SOP). For transparency reasons, it should be possible to make the SOP available to other interested parties upon request.

> The SOP should specify:

Reviewer: ```suggestion
The SOPs should specify:
```

* The range of possible review outcomes (e.g.: approval, refusal, revision);

* The type of consensus required for review decisions (e.g.: unanimity, majority vote, quorum);

* The process of a criteria-based access review (this is complemented by the LOC below);

* The process for submitting an appeal or a revised application after a negative decision;

* The method of compiling statistics regarding KPIs (e.g.: number of applications approved/rejected, average time taken);

* The method for reviewing decisions in order to improve DAC performance;

* The process of modifying the TOR, SOP, and LOC (e.g.: How can members suggest and make amendments?).

### List of Criteria (LOC) for Data Access Decisions

Each DAC should establish a list of criteria (LOC) that govern the review process and harmonise this process across Data Access Requests. The LOC should be documented and subject to change through a predefined process (see last bullet point of the SOP). For transparency reasons, it should be possible to make the LOC available to other interested parties upon request.

Access requests should be checked against these items:

> 1. The proposed research has received a positive ethics vote or an ethics approval from a credible Institutional Review Board (Research Ethics Committee), if applicable.

Reviewer: We previously used 'Institutional Review Board | Research Ethics Committee'
